Tests/UnitTests/Mocks/MockETagManager.swift

```diff
@@ -13,25 +13,39 @@ import Foundation
 // swiftlint:disable type_name
 class MockETagManager: ETagManager {
 
+    struct ETagHeaderRequest {
+        var urlRequest: URLRequest
+        var refreshETag: Bool
+        var signatureVerificationEnabled: Bool
```

Comment:
Same here, could these be let? I don't think callers of this mock would need to modify this?

Reply:
Ditto, it doesn't make a difference, it would only be mutable for a `var ETagHeaderRequest` anyway.